dev/env/scripts/lib.sh

```diff
@@ -49,6 +49,12 @@ dump_env() {
 init() {
     set -eu -o pipefail
 
+    kubectl cluster-info  | grep -E "stage|prod|openshiftapps"
+    if [[ "$?" -eq 0 ]]; then
+        echo "Executing against prod or staging not allowed. Please check your kubectl conetxt."
```

Comment:
Typo. 
Side comment: Will errors to execute kubectl cluster-info always be due to a permission error (`not allowed`) ? 
```suggestion
        echo "Executing against prod or staging not allowed. Please check your kubectl context."
```

Reply:
Fixed